src/main/java/seedu/address/model/person/MatriculationNumber.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's matriculation number in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidMatriculationNumber(String)}
+ */
+public class MatriculationNumber {
+
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Matriculation number should be in the following format: A1234567X, where X is a placeholder";
```

Comment:
im a bit confused by the "where x is a placeholder" statement

Reply:
Updated accordingly to reflect X is a placeholder for the last alphabet of the student's matriculation number